Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM smartcosmos/java
+MAINTAINER The SMART COSMOS Authors
```

Comment:
Hmm... Maybe there is a nicer name. Somewhere I saw something like "SMART COSMOS Platform Core Team".